modules/ROOT/pages/versioning.adoc

```diff
@@ -0,0 +1,17 @@
+= Versioning
+:description: The approach to versioning of the Neo4j GraphQL Library.
```

Comment:
Is it worth mentioning the LTS vs latest approach?

Currently version 5 is LTS, which is still maintained but no new features are added, whilst version 7 (recommended) is actively being worked on